### PR TITLE
Small improvements to Vertex pipeline generation

### PR DESCRIPTION
**Pull request by szczeles**
#### Description

Small improvements to Vertex pipeline generation
- passing env in kedro commands
- setting proper artifact type based on layer in catalog

##### PR Checklist
- [x] Tests added
- [x] [Changelog](CHANGELOG.md) updated

### CODE REVIEW

File: tests/test_vertex_ai_generator.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def test_should_skip_volume_init_if_requested(self):
             pipeline()
 
         # then
+        print(len(dsl_pipeline.ops))
```
Comment (michalzelechowski-getindata) [inline tests/test_vertex_ai_generator.py:54]: Probably leftover.

File: kedro_kubeflow/vertex_ai/io.py [hunk 2]
```diff
@@ -54,6 +54,12 @@ def is_file_path_input(input_data):
     return input_params, input_specs
 
 
+def get_output_type(output, catalog):
+    if catalog[output].get("layer") == "models":
```
Comment (michalzelechowski-getindata) [inline kedro_kubeflow/vertex_ai/io.py:58]: Am I correct, that this fragment picks also mlflow artifacts if these are stored locally? How does it work if artifacts are stored in remote mlflow server?

File: tests/test_vertex_ai_generator.py (hunk 1 above)
Comment (szczeles) [reply tests/test_vertex_ai_generator.py]: Thanks! Dropped

File: kedro_kubeflow/vertex_ai/io.py (hunk 2 above)
Comment (szczeles) [reply kedro_kubeflow/vertex_ai/io.py:58]: That's right, this part does not differentiate between remote and local artifacts. Unfortunately I wasn't able to test the mlflow artifacts yet due to connectivity issues, but as soon as I overcome these, I will update the code to support these somehow

**Review by michalzelechowski-getindata — APPROVED** [review]

**Pull request merged**